src/include/rados/librados.hpp

```diff
@@ -10,6 +10,7 @@
 #include <utility>
 #include "memory.h"
 #include "buffer.h"
+#include "common/ceph_time.h"
```

Comment:
buffer.h was a mistake - it should have hidden a lot more so we would be able to change it more easily without breaking librados users